frontend/public/components/monitoring/alert-manager-config.tsx

```diff
@@ -97,7 +269,25 @@ type AlertManagerConfigurationProps = {
   onCancel?: () => void;
 };
 
-type AlertRoutingProps = {
+type AlertManagerProps = {
   config: any;
```

Comment:
Would it be worth defining a type for `config` to specify which fields we expect it to have?

Reply:
Hmm...that would be very large:  https://prometheus.io/docs/alerting/configuration/#configuration-file

Reply:
updated `AlertManagerProps`

Reply:
👍